plutus-numeric/src/PlutusTx/Numeric/Extra.hs

```diff
@@ -410,19 +455,18 @@ infixr 8 ^
 
  @since 4.2
 -}
-{-# INLINEABLE scaleNat #-}
+{-# DEPRECATED scaleNat "Inefficient. Use 'semiscale' instead" #-}
+{-# INLINEABLE [2] scaleNat #-}
```

Comment:
You can drop the `[2]` on that. This is a remnant of my attempt to make rewrites work.